datm/cime_config/namelist_definition_datm.xml

```diff
@@ -38,7 +38,7 @@
         1PT.urbanc_alpha
       </value>
       <value datm_mode="1PT" model_grid="CLM_USRDAT" neon="True">
-        NEON.$NEONSITE
+        NEON.$NEONSITE,NEON.$NEONSITE.PRECIP
```

Comment:
I propose that you call the PRISM precip data for NEON

NEON.$NEONSITE.PRISM_PRECIP

Having it generic "PRECIP" makes it sound like its preceip data that would always be used.

Reply:
Great idea, thanks!